util.go

```diff
@@ -88,12 +88,14 @@ func convertAtTimesToDateTime(atTimes AtTimes, location *time.Location) ([]time.
 		}
 		atTimesDate = append(atTimesDate, at.time(location))
 	}
-	slices.SortStableFunc(atTimesDate, func(a, b time.Time) int {
-		return a.Compare(b)
-	})
+	slices.SortStableFunc(atTimesDate, AscendingTime)
 	return atTimesDate, nil
 }
 
+func AscendingTime(a, b time.Time) int {
```

Comment:
This shouldn't need to be exported

Reply:
good catch, I didn't realize that it was all in the same package. Pushed :+1: